### DISCUSSION TITLE
Vercel does not rebuilt after being successfully triggered via Deploy Hook

### DISCUSSION THREAD

**Discussion by ronaldrupp** [vercel/community#1710] [Help] [answered]
### Summary

I have connected my Prismic Repo with the Vercel Project via a Deploy Hook but suddenly it does not trigger a rebuilt of the site. Does anyone have a clue?

**Comment by ronaldrupp** [comment, marked as answer]
I set a cron job to "*/5 * * * *" which is not included in the hobby plan, that's why

**Reply by amyegan** [reply]
Thanks for coming back to share this, Ronald! I also want to share a few documentation links for others who run into this question:
- [Limits](https://vercel.com/docs/cron-jobs#limits) (Hobby accounts get 2 cron jobs that can be triggered once a day)
- [Hobby scheduling limits](https://vercel.com/docs/cron-jobs#hobby-scheduling-limits)
- [How long can a cron job run?](https://vercel.com/docs/cron-jobs#how-long-can-a-cron-job-run)
